awx/main/tasks.py

```diff
@@ -41,6 +43,10 @@
 # Django-CRUM
 from crum import impersonate
 
+# GitPython
+import git
```

Comment:
I think I may need some help understanding what is going on in this file.... is it just Runner usage cleanup?